backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -1,20 +1,51 @@
 import json
+import random
+import string
 
 import pytest
 import requests
 import sseclient
+import transaction
 
 from tracim_backend.error import ErrorCode
 from tracim_backend.lib.core.live_messages import LiveMessagesLib
+from tracim_backend.models.data import Content
+from tracim_backend.models.revision_protection import new_revision
 from tracim_backend.tests.fixtures import *  # noqa: F403,F40
 
 
+@pytest.fixture
+def big_html_document(workspace_api_factory, content_api_factory, session) -> Content:
+    workspace_api = workspace_api_factory.get()
+    workspace = workspace_api.create_workspace(label="Foobar")
+    content_api = content_api_factory.get()
+
+    description = "".join(random.choice(string.ascii_letters) for _ in range(2000000))
```

Comment:
I don't think we should have randomness in tests. Is it sufficient to generate a zeroed string, or provide a seed so we are sure the test is always the same / deterministic? Not that it should impact anything here, but still.

Reply:
You're right, I'll change this to a fixed content.

Reply:
I suggest to also put UFT-8 chars...

Reply:
After discussion, we are putting this for future work to ease the release of Tracim 3.0.0. Unexpectedly, putting Japanese characters in there makes the test ruuuuunnnn veeerrrryyy sloooowly.